CHANGELOG.md

```diff
@@ -14,6 +14,15 @@ The format is based on [Keep a Changelog].
 > - **Security**: in case of vulnerabilities.
 
 
+## [UNRELEASED]
+
+### Changed
+
+- The Exception hierarchy has been refined with more specialized classes - 
+  please note it is still possible to continue to catch their parent 
+  exceptions, however. Also, the exception class `IBMQApiUrlError` was 
+  removed. (\#480)
```

Comment:
nit, but can you give a quick example about the parent exception, and the replacement of `IBMQApiUrlError`?  Maybe something like
```
The Exception hierarchy has been refined with more specialized classes. You can, however, continue to catch their parent exceptions (such as `IBMQAccountError`). Also, the exception class `IBMQApiUrlError` has been replaced by `IBMQAccountCredentialsInvalidUrl` and `IBMQAccountCredentialsInvalidToken`.
```